python/ray/_private/usage/usage_lib.py

```diff
@@ -210,13 +341,19 @@ def get_cluster_metadata(gcs_client, num_retries) -> dict:
 
 
 def generate_report_data(
-    cluster_metadata: dict, total_success: int, total_failed: int, seq_number: int
+    cluster_metadata: dict,
+    cluster_config: dict,
+    total_success: int,
+    total_failed: int,
+    seq_number: int,
 ) -> UsageStatsToReport:
     """Generate the report data.
 
     Params:
         cluster_metadata (dict): The cluster metadata of the system generated by
             `_generate_cluster_metadata`.
+        cluster_config (dcit): The cluster (autoscaler) config generated by
```

Comment:
```suggestion
        cluster_config (dict): The cluster (autoscaler) config generated by
```

Reply:
Should we make it a dataclass?

Reply:
There should be a Pydantic model for cluster configs, but that's definitely outside of the scope of this PR.

Reply:
Wait cluster config is something different here.

Reply:
+1 for dataclass, also call it something other than cluster config as mentioned above